pulsar-broker/src/test/java/org/apache/pulsar/broker/service/ReplicatorTest.java

```diff
@@ -1315,9 +1315,6 @@ public void testDoNotReplicateSystemTopic() throws Exception {
             Assert.assertEquals(admin2.topics().getStats(systemTopic).getReplication().size(), 0);
             Assert.assertEquals(admin3.topics().getStats(systemTopic).getReplication().size(), 0);
         });
-        cleanup();
-        setup();
-
```

Comment:
This seems to be unrelated to this change.

Reply:
restored